notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Re: the comment about how to talk about functions, we can add that to a "Prerequisites" section and link to a resource if necessary, e.g. https://www.py4e.com/html3/04-functions</p><p> </p><p>Adding: scientific concepts can go there too, like Periodograms. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Also, cna this part be moved to later? It relates to the flare dataset but that's not discussed until the end</p>

Reply:
<p>I really like having a prerequisites section! We could also put the flares notebook in there too? And you're definitely right, this can be moved to later.</p>